src/App.js

```diff
@@ -1,16 +1,17 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
-import Button from 'react-bootstrap/Button'
+import Container from 'react-bootstrap/Container'
 import Header from './stuff/Header';
 import Footer from './stuff/Footer';
 import Main from './stuff/Main';
```

Comment:
So far this project is lookin' awesome!

Feedback here for two things: 
- Consider renaming the folder `stuff` to something descriptive(we want to try and name it something a bit more helpful)
- For PR name, we want to stick to industry standards as much as possible. Try your best to keep it short and simple like you did here, but instead name it something that relates to the overall goal, or features for today (you displayed images, allowed users to vote and made it responsive, so maybe something like: Displays beast data and votes with Bootstrap works well...)